contracts/facets/EmbalmerFacet.sol

```diff
@@ -136,6 +141,15 @@ contract EmbalmerFacet {
             archaeologistAddresses[i] = archaeologists[i].archAddress;
         }
 
+        // If the storage fee is 0, then the storage fee was never set since the
+        // default value is 0. This means that either the arweave archaeologist
+        // was not included in the list of archaeologists or the arweave
+        // archaeologist set their storage fee to 0. In either case the
+        // transaction should be reverted.
+        if (storageFee == 0) {
+            revert LibErrors.ArweaveArchaeologistNotInList();
+        }
```

Comment:
The only edge case here would be if an archaeologist set their `FEE_PER_BYTE` value to 0, thus making an arweave TX free for the embalmer (although the arch would still pay the arweave TX fee when they upload the file)

We may need to require this `FEE_PER_BYTE` *not* be 0 for the archaeologist config, or make the front-end still pay some insignificant amount (e.g. 0.000001 SARCO) in the event `FEE_PER_BYTE` is set to 0.

Reply:
I was also thinking that the archaeologist service would validate this number is not 0. We should probably set the default FEE_PER_BYTE value to a realistic number in the arch service.